catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/parser/ParserTestStringFunctions.java

```diff
@@ -185,6 +185,89 @@ public void testJoin() {
 				+ Double.NaN + Double.NaN, testSprite);
 	}
 
+	@Test
+	public void testJoin3() {
+		String firstParameter = "first";
+		String secondParameter = "second";
+		String thirdParameter = "third";
+		FormulaEditorTestUtil.testTripleParameterFunction(Functions.JOIN3, InternTokenType.STRING,
+				firstParameter,
+				InternTokenType.STRING, secondParameter,
+				InternTokenType.STRING, thirdParameter,
+				firstParameter + secondParameter + thirdParameter,
+				testSprite);
+
+		firstParameter = "";
+		secondParameter = "second";
+		thirdParameter = "third";
+		FormulaEditorTestUtil.testTripleParameterFunction(Functions.JOIN3, InternTokenType.STRING,
+				firstParameter,
+				InternTokenType.STRING, secondParameter,
+				InternTokenType.STRING, thirdParameter,
+				firstParameter + secondParameter + thirdParameter,
+				testSprite);
+
+		firstParameter = "first";
+		secondParameter = "";
+		thirdParameter = "third";
+		FormulaEditorTestUtil.testTripleParameterFunction(Functions.JOIN3, InternTokenType.STRING,
+				firstParameter,
+				InternTokenType.STRING, secondParameter,
+				InternTokenType.STRING, thirdParameter,
+				firstParameter + secondParameter + thirdParameter,
+				testSprite);
+
+		firstParameter = "first";
+		secondParameter = "";
+		thirdParameter = "";
+		FormulaEditorTestUtil.testTripleParameterFunction(Functions.JOIN3, InternTokenType.STRING,
+				firstParameter,
+				InternTokenType.STRING, secondParameter,
+				InternTokenType.STRING, thirdParameter,
+				firstParameter + secondParameter + thirdParameter,
+				testSprite);
+
+		firstParameter = "33";
+		secondParameter = "44";
+		thirdParameter = "55";
+		FormulaEditorTestUtil.testTripleParameterFunction(Functions.JOIN3, InternTokenType.NUMBER,
+				firstParameter,
+				InternTokenType.NUMBER, secondParameter,
+				InternTokenType.NUMBER, thirdParameter,
+				firstParameter + secondParameter + thirdParameter,
+				testSprite);
+
+		firstParameter = "5*3-6+(8*random(1,2))";
+		secondParameter = "string'**##!§\"$\'§%%/&%(())??";
```

Comment:
```suggestion
		secondParameter = "string'**##!§\"$\'§%\"%/&%(())??";
```
Let's check " as well. Should be OK, but maybe needs \\\" instead of just \"